pallets/claims/src/lib.rs

```diff
@@ -554,10 +558,15 @@ impl<T: Config> Pallet<T> {
 
 	// Constructs the message that PolkadotJS would sign.
 	fn polkadotjs_signable_message(what: &[u8], extra: &[u8]) -> Vec<u8> {
+		let mut v = Vec::new();
+		let polkadotjs_prefix = b"<Bytes>";
+		let polkadotjs_suffix = b"</Bytes>";
```

Comment:
I believe there's been a misunderstanding, @shekohex. Our task is to wrap the `<Bytes></Bytes>` tags around the hashed message, which is the result of `keccak_256`. 

On the UI side, we hash the payload before sending it to the Polkadot wallet extension for signing. Prior to signing, the wallet automatically wraps the hashed payload with the `<Bytes></Bytes>` tags. 

Therefore, I propose that we should follow the same practice on the backend, wrapping the hashed message with the `<Bytes></Bytes>` tags.

Reply:
@AtelyPham can you make the necessary changes you're requesting? Full stack approach will make this move faster.

Reply:
Consider updating [this](https://github.com/webb-tools/tangle/blob/47964dd838a3e29ad405649476666abac94bc2d5/pallets/claims/src/lib.rs#L590C3-L590C3) to the following:

```rust

match sr25519_verify(&s.0, &msg, &public) {
	true => Some(addr),
	false => {
		// If the signature verification fails, we try to wrap the hashed msg in a
		// `<Bytes></Bytes>` tag and try again.
		let polkadotjs_prefix = b"<Bytes>";
		let polkadotjs_suffix = b"</Bytes>";

		let mut wrapped_msg = Vec::new();
		wrapped_msg.extend_from_slice(polkadotjs_prefix);
		wrapped_msg.extend_from_slice(&msg);
		wrapped_msg.extend_from_slice(polkadotjs_suffix);

		match sr25519_verify(&s.0, &wrapped_msg, &public) {
			true => Some(addr),
			false => None,
		}
	},
}

```

Please note that it isn't necessary to wrap the message in a `<Bytes></Bytes>` tag prior to hashing.

Reply:
Will make the changes and push it.